utils/copy-flow-source.js

```diff
@@ -0,0 +1,17 @@
+const flowCopySource = require("flow-copy-source");
+const path = require("path");
+const fs = require("fs");
+
+const packages = fs.readdirSync(path.join(__dirname, "../packages"));
+
+for (const pkg of packages) {
+    // eslint-disable-next-line
```

Comment:
🛑 Would be good to include the rule that is being blocked, rather than blocking all errors.